src/services/CleanupService.ts

```diff
@@ -361,26 +361,33 @@ export const clearCookiesForThisDomain = async (
       );
       if (r) cookieDeletedCount += 1;
     }
-    showNotification({
+    showNotification(
```

Comment:
Hmm, it might be a good idea to decouple some of the notification logic away from Cleanup service. It can be a wrapper function around `showNotification` that takes in something that Cleanup service would return (maybe CleanReasonObject with some refactoring to include the other side data hostnames). Based on whats in that object, you can output a different types of notifications. Though this would require some refactoring that I don't expect in this PR.